src/2_RTB/NormeDiProgetto/content.typ

```diff
@@ -34,6 +34,81 @@ I _Feature branch_ vengono aperti a partire dalle issue create nell' _Issue Trac
 
 Nel caso del repository _InnovaCity_ il branch _main_ viene utilizzato per la pubblicazione di cambiamenti _major_, cioè quando si è implementato un numero di funzionalità significativo all'avanzamento del progetto. In quel caso è il responsabile che esegue l'approvazione finale.
 
+
+=== Analisi dei Requisiti
+
+
+L’analisi dei requisiti, condotta dagli Analisti, rappresenta un’attività preliminare nello sviluppo di un sistema software. Questa attività mira a definire le funzionalità che il nuovo prodotto deve offrire. L'obiettivo principale è quello di identificare, in modo chiaro e completo, i requisiti che il software sviluppato deve soddisfare, al fine di rispondere adeguatamente alle esigenze degli utenti e della Proponente.
+
+L'analisi si compone di varie fasi:
+
+  - scopo del prodotto: si devono soddisfare le esigenze della Proponente.
+  - definizione degli attori: entità o persone che interagiscono con il software.
+  - definizione dei casi d'uso: rappresentazione di uno scenario specifico che descrive come un attore interagisce con il software.
+  - definizione di requisiti obbligatori, opzionali, di vincolo.
+  - confronti interni ed esterni.
+
+L'interazione con la Proponente è indispensabile per una precisa interpretazione ed espansione dettagliata della realizzazione.
```

Comment:
Potrebbe essere formulato meglio, tipo "La collaborazione con la Proponente è essenziale per comprendere in modo accurato e approfondire dettagliatamente il processo di realizzazione del prodotto."